core/iwasm/interpreter/wasm_loader.c

```diff
@@ -4591,7 +4591,7 @@ wasm_loader_ctx_destroy(WASMLoaderContext *ctx)
 }
 
 static WASMLoaderContext *
-wasm_loader_ctx_init(WASMFunction *func)
+wasm_loader_ctx_init(WASMFunction *func, char *error_buf, uint32 error_buf_size)
 {
     WASMLoaderContext *loader_ctx =
         loader_malloc(sizeof(WASMLoaderContext), NULL, 0);
```

Comment:
Use `loader_malloc(sizeof(WASMLoaderContext), error_buf, error_buf_size)`, and no need to add line 4642. Same as following loader_malloc.

Reply:
Done, thanks~